app/models/guidance_page.rb

```diff
@@ -0,0 +1,24 @@
+class GuidancePage
+  def initialize(path, content: nil)
+    @path = path
+    @content = content
+  end
+
+  def sub_headings
+    headings = page_contents.scan(/^##\s(.*)$/).map(&:first)
+
+    headings.index_by { |heading| heading.underscore.parameterize.gsub("_", "-") }
```

Comment:
`parameterize(separator: "-")`

Reply:
ah sorry I wasn't very clear, I just meant you could drop the `gsub` in favour of specifying `separator` to the `parameterize` call, so this should be equivalent but without the need for the extra `gsub`:

```
headings.index_by { |heading| heading.underscore.parameterize(separator: "-") }
```

Reply:
oh I follow now - ignore me, I must need a coffee 😄